test/integration/select_test.lua

```diff
@@ -217,32 +217,32 @@ pgroup.test_negative_first = function(g)
     t.assert_equals(objects, helpers.get_objects_by_idxs(customers, {2, 3, 4}))
 
     -- id >= 2
-    -- first -2 after 5
+    -- first -4 after 5
```

Comment:
I think it's better to leave a more human-readable version, like "last 4"

Reply:
I don't see any value in these comments at all. This is just a duplication of the code below. I just copied the nearest code to make it look the same (you can see the same comments in the whole file).